lib/fileutils.js

```diff
@@ -43,3 +68,29 @@ function which(exe, cb){
     cb(!!exePath)
   })
 }
+
+exports.findableByWhichOrModule = findableByWhichOrModule
+function findableByWhichOrModule(cb){
+  var browser = this
+
+  findableBy(which).call(browser, function(findable){
+    if(findable){
+      cb(findable)
+    }else{
+      findableByModule.call(browser, cb)
+    }
+  })
+}
+
+exports.findableByWhereOrModule = findableByWhereOrModule
+function findableByWhereOrModule(cb){
```

Comment:
Could you merge the common parts with `findableByWhichOrModule`? Both look nearly the same and it is difficult to spot the difference.